steward/src/commands/keys/cosmos/add.rs

```diff
@@ -6,24 +6,28 @@ use rand_core::OsRng;
 use signatory::FsKeyStore;
 use std::path;
 
+/// Steward keys cosmos add [name]
 #[derive(Command, Debug, Default, Parser)]
+#[clap(
+    long_about = "DESCRIPTION \n\n Create a new Cosmos Key.\n This command creates a new Cosmos key. It has an overwrite option, which if set to true, overwrites\n an existing key in the keystore with similar keyname."
```

Comment:
Saying the same thing twice in the first two lines here.

Reply:
Edited the command description